src/components/ProductTable.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Product from './Product';
+import '../styles/productTable.css';
+
+const PRODUCTS = [
+  { name: 'Apple', price: '$0.79', stocked: true },
+  { name: 'Banana', price: '$0.50', stocked: true },
+  { name: 'Peach', price: '$0.85', stocked: false },
+  { name: 'Watermelon', price: '$1.50', stocked: true },
+  { name: 'Mango', price: '$2.00', stocked: false }
+];
+
+class ProductTable extends React.Component {
```

Comment:
```suggestion
const ProductTable = (props) => {
```
Use functional components over class components. This also means you'll need to remove the render function. The component will render whatever is returned.